addons/project/models/project.py

```diff
@@ -1591,6 +1608,23 @@ def copy_data(self, default=None):
         defaults = super().copy_data(default=default)
         return [{k: v for k, v in default.items() if k in self.SELF_READABLE_FIELDS} for default in defaults]
 
+    @api.model
+    def new(self, values={}, origin=None, ref=None):
+        # Required in case we create a new subtask
+        # as we will update the cache on res.users
+        # and we won't be able to filter the active
+        # users at that time
+        if origin and 'user_ids' in values and self.env.user.has_group('base.group_portal'):
+            self._ensure_fields_are_accessible(values, check_group_user=False)
+            if issubclass(type(origin), self.pool['project.task']):
+                task = origin
+            else:
+                task = self.browse(origin)
```

Comment:
So overcomplicated.  `origin` is either a record (same type as `self`) or `None`, so...
```suggestion
            task = origin or self.browse()
```

Reply:
Indeed, thanks ! 🙂